ansible_collections/arista/avd/plugins/action/verify_requirements.py

```diff
@@ -31,34 +31,52 @@
 MIN_PYTHON_SUPPORTED_VERSION = (3, 8)
 
 
-def _validate_python_version(result: dict) -> bool:
+def _validate_python_version(info: dict, result: dict) -> bool:
     """
     TODO - avoid hardcoding the min supported version
 
+    Args:
+      info (dict): Dictionary to store information to present in ansible logs
+      result (dict): Module result dictionary to store deprecation warnings
+
     return False if the python version is not valid
     """
-    result["python_version_info"] = {
+    info["python_version_info"] = {
         "major": sys.version_info.major,
         "minor": sys.version_info.minor,
         "micro": sys.version_info.micro,
         "releaselevel": sys.version_info.releaselevel,
         "serial": sys.version_info.serial,
     }
-    result["python_path"] = sys.path
+    info["python_path"] = sys.path
 
+    running_version = ".".join(str(v) for v in sys.version_info[:3])
+    min_version = ".".join(str(v) for v in MIN_PYTHON_SUPPORTED_VERSION)
     if sys.version_info < MIN_PYTHON_SUPPORTED_VERSION:
-        running_version = ".".join(str(v) for v in sys.version_info[:3])
-        min_version = ".".join(str(v) for v in MIN_PYTHON_SUPPORTED_VERSION)
         display.error(f"Python Version running {running_version} - Minimum Version required is {min_version}", False)
         return False
+    elif sys.version_info[:2] == MIN_PYTHON_SUPPORTED_VERSION:
+        result.setdefault("deprecations", []).append(
+            {
+                "msg": (
+                    f"You are currently running Python {running_version}. The next minor release of AVD  after November 6th 2023 will drop support for Python"
+                    f" {min_version} as it will be dropping support for ansible-core<2.14 and ansible-core>=2.14 does not support Python {min_version} as"
+                    " documented here: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix"
```

Comment:
```suggestion
                    f"You are currently running Python {running_version}. The next minor release of AVD after November 6th 2023 will drop support for Python"
                    f" {min_version} as it will be dropping support for ansible-core<2.14 and ansible-core>=2.14 does not support Python {min_version} as"
                    " documented here: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix"
```